### PR TITLE
Test if AWS supported region url exists

### PR DESCRIPTION
**Pull request by joyc-bq**
Test that link is working in this error message: 

 IamAuthConnectionPlugin.unsupportedRegion=Unsupported AWS region ''{0}''. For supported regions, please read https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html

### CODE REVIEW

File: wrapper/src/main/java/software/amazon/jdbc/plugin/AwsSecretsManagerConnectionPlugin.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ public AwsSecretsManagerConnectionPlugin(PluginService pluginService, Properties
     final Region region = Region.of(regionString);
     if (!Region.regions().contains(region)) {
       throw new RuntimeException(Messages.get(
-          "AwsSecretsManagerConnectionPlugin.unsupportedRegion",
+          "IamAuthConnectionPlugin.unsupportedRegion",
```
Comment (karenc-bq) [inline wrapper/src/main/java/software/amazon/jdbc/plugin/AwsSecretsManagerConnectionPlugin.java:123]: I propose we change the message name to something more generic, maybe AwsSdk.unsupportedRegion or AwsAuth.unsupported...? A bit weird to have an IamAuthConnPlugin message in the secrets manager plugin.

File: wrapper/src/test/java/software/amazon/jdbc/plugin/IamAuthConnectionPluginTest.java [hunk 2]
```diff
@@ -193,4 +207,6 @@ private void testGenerateToken(
     assertEquals(GENERATED_TOKEN, PropertyDefinition.PASSWORD.getString(props));
     assertEquals(GENERATED_TOKEN, IamAuthConnectionPlugin.tokenCache.get(PG_CACHE_KEY).getToken());
   }
+
+
```
Comment (karenc-bq) [inline wrapper/src/test/java/software/amazon/jdbc/plugin/IamAuthConnectionPluginTest.java:211]:
```suggestion
```

File: wrapper/src/test/java/software/amazon/jdbc/plugin/IamAuthConnectionPluginTest.java [hunk 3]
```diff
@@ -154,6 +157,17 @@ public void testConnectWithSpecifiedHost() throws SQLException {
         "pg.testdb.us-east-2.rds.amazonaws.com");
   }
 
+  @Test
+  public void testAwsSupportedRegionsUrlExists() throws IOException {
+    URL url =
```
Comment (karenc-bq) [inline wrapper/src/test/java/software/amazon/jdbc/plugin/IamAuthConnectionPluginTest.java:162]: please add finals

File: wrapper/src/test/java/software/amazon/jdbc/plugin/IamAuthConnectionPluginTest.java [hunk 4]
```diff
@@ -154,6 +157,17 @@ public void testConnectWithSpecifiedHost() throws SQLException {
...
+        new URL("https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html");
+    HttpURLConnection urlConnection = (HttpURLConnection) url.openConnection();
+    urlConnection.setRequestMethod("HEAD");
```
Comment (karenc-bq) [inline wrapper/src/test/java/software/amazon/jdbc/plugin/IamAuthConnectionPluginTest.java:165]: Is this required?

Comment (joyc-bq) [reply wrapper/src/test/java/software/amazon/jdbc/plugin/IamAuthConnectionPluginTest.java]: it improves response time, but for just one get request it isn't needed. removed

**Comment by davecramer** [comment]
cool, thanks

**Review by karenc-bq — APPROVED** [review]

**Pull request merged**